tests/parameter_handler/parameter_handler_read_json_05.cc

```diff
@@ -0,0 +1,51 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2020 - 2024 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+
+// Like parameter_handler_read_json_05 but output with
+// ParameterHandler::OutputStyle::KeepChanged.
```

Comment:
I would maybe make the comment here a bit more verbose to describe the content of the test.
```suggestion
// Like parameter_handler_read_json_05 but output with
// ParameterHandler::OutputStyle::KeepChanged. This is achieved by adding two parameters with defualt values, altering one of the value of the parameters and printing the parameters using the ParameterHandler::OutputStyle::KeepChanged.
```